dashboard/app/views/levels/editors/fields/_blockly.html.haml

```diff
@@ -43,6 +43,9 @@
       %p Number is a multiplier for how long each step takes (so higher numbers are slower). Default is 5 for Maze, 2 for Bee.
       = f.number_field :step_speed
 
+    .field
+      = render partial: 'levels/editors/fields/checkboxes', locals: {f: f, field_name: :include_shared_functions, description: "Make shared functions and behaviors available"}
```

Comment:
this looks like it makes share functions and behaviors visible for all blockly level types. behaviors sound specific to spritelab, so maybe this is not correct?